packages/@snowpack/plugin-babel/README.md

```diff
@@ -25,3 +28,4 @@ npm install --save-dev @snowpack/plugin-babel
 | Name    | Type       | Description                                                                                                                                        |
 | :------ | :--------- | :------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `input` | `string[]` | (optional) By default, Babel scans & transfoms these extensions: `['.js', '.jsx', '.ts', '.tsx']`. Modify this array if you’d like to change this. |
+| `transformOptions` | `object` | (optional) See [https://babeljs.io/docs/en/options](https://babeljs.io/docs/en/options) |
```

Comment:
What are your thoughts on _not_ having a `transformOptions` namespace, and instead just passing all other options straight to Babel? e.g.:

```js
const { input, ...babelOptions = {} } = options;

babel.transformFileAsync(filePath, { ...babelOptions });
```

I think this would have the benefit of being more predictable / simpler to use, e.g.:

```json
{
  "plugins": [["@snowpack/plugin-babel", { "configFile": "./babel.config.js" }]]
}
```

Reply:
Maybe need to add other plugin options will cause duplication